Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -106,6 +106,17 @@ static bool experimentalFeatureEnabled(const String& key, bool defaultValue = fa
     return defaultValue;
 }
 
+static NSString* applicationOrProcessIdentifier()
+{
+    NSString *identifier = [NSBundle mainBundle].bundleIdentifier;
+    NSString *processName = [NSProcessInfo processInfo].processName;
+    if ([identifier isEqualToString:@"com.apple.Safari"] && [processName isEqualToString:@"SafariForWebKitDevelopment"])
```

Comment:
Would be nice to add a comment to explain why we're special casing SafariForWebKitDevelopment here.